src/components/cards/Hand.tsx

```diff
@@ -5,10 +5,14 @@ import { HandContainer, StyledCard } from "./CardElements";
 import ReversableCard from "./ReversableCard";
 
 import useGameContext from "../../hooks/useGameContext";
+
+import pointsHelpers from "../points/PointsHelper";
+
 import { rules } from "../../const/rules";
+import { GameState } from "../../const/gameState";
 
 export default function Hand({ player = false }) {
```

Comment:
Rozbij ten komponent na dwa, osobno dla playera i croupiera np

```suggestion
export default function CroupierHand() {
  const { croupierCards, currentRoundStatus} = useGameContext();

  const isReversed = currentRoundStatus=== GameState.CROUPIER_ROUND;

  return (
    <HandContainer>
      {croupierCards.map((card, index) => {
        if (currentTurn === TurnStatus.player && index === 1) {
          return <ReversableCard key={index} aversImage={card.image} isReversed={isReversed} />;
        }

        return <StyledCard key={index} image={card.image} />;
      })}
    </HandContainer>
  );
}


export default function PlayerHand() {
  const { playerCards } = useGameContext();

  return (
    <HandContainer>
      {playerCards.map((card, index) => {
        return <StyledCard key={index} image={card.image} />;
      })}
    </HandContainer>
  );
}

```